<ui version="4.0" >
  <author>GeometryFactory</author>
  <class>cgal_cg</class>
  
 <widget class="QMainWindow" name="cgal_cg" >
 
  <property name="geometry" >
    <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
    </rect>
  </property>
  <property name="windowTitle" >
    <string>CGAL Delaunay Triangulation</string>
  </property>
  <property name="windowIcon" >
    <iconset resource="../resources/CGAL.qrc" >
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
   
  <widget class="QWidget" name="centralwidget" >
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="QGraphicsView" name="graphicsView" >
          <property name="focusPolicy" >
            <enum>Qt::StrongFocus</enum>
          </property>
          <property name="transformationAnchor" >
            <enum>QGraphicsView::NoAnchor</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>

    <widget class="QToolBar" name="toolBar" >
      <property name="windowTitle" >
        <string>Visualization Tools</string>
      </property>
      <attribute name="toolBarArea" >
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak" >
        <bool>false</bool>
      </attribute>
      <addaction name="actionShowDelaunay" />
      <addaction name="actionShowVoronoi" />
      <addaction name="actionShowConvexHull"/>
    </widget>

   <action name="actionShowDelaunay" >
     <property name="checkable" >
       <bool>true</bool>
     </property>
     <property name="checked" >
       <bool>false</bool>
     </property>
     <property name="icon" >
       <iconset resource="../icons/Triangulation_2.qrc" >
         <normaloff>:/cgal/Triangulation_2/Delaunay_triangulation_2.png</normaloff>:/cgal/Triangulation_2/Delaunay_triangulation_2.png
       </iconset>
     </property>
     <property name="text" >
       <string>Show &amp;Delaunay Triangulation</string>
     </property>
     <property name="shortcut" >
       <string>Ctrl+D</string>
     </property>
   </action>

    <action name="actionShowVoronoi" >
      <property name="checkable" >
        <bool>true</bool>
      </property>
      <property name="checked" >
        <bool>false</bool>
      </property>
      <property name="icon" >
        <iconset resource="../icons/Triangulation_2.qrc" >
        <normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</iconset>
      </property>
      <property name="text" >
        <string>Show &amp;Voronoi Diagram</string>
      </property>
      <property name="shortcut" >
        <string>Ctrl+V</string>
      </property>
    </action>
   
    <action name="actionShowConvexHull">
      <property name="checkable" >
        <bool>true</bool>
      </property>
      <property name="checked" >
        <bool>false</bool>
      </property>
      <property name="icon" >
        <iconset resource="../icons/Triangulation_2.qrc" >
          <normaloff>:/cgal/Triangulation_2/Convex_hull.png</normaloff>:/cgal/Triangulation_2/Convex_hull.png
        </iconset>
      </property>
      <property name="text" >
        <string>Show &amp;Convex Hull</string>
      </property>
      <property name="shortcut" >
        <string>Ctrl+C</string>
      </property>
    </action>
   
 </widget>
  
 <resources>
  <include location="Delaunay_triangulation_2.qrc" />
  <include location="../icons/File.qrc" />
  <include location="../resources/CGAL.qrc" />
  <include location="../icons/Triangulation_2.qrc" />
  <include location="../icons/Input.qrc" />
 </resources>
 <connections/>
</ui>
